geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -313,27 +305,28 @@ public String toString() {
     return null;
   }
 
-  private double processIncrement(String stringIncr) {
-    double incr;
-    switch (stringIncr) {
+  private double processIncrement(byte[] value) {
```

Comment:
Since this method now processes scores and increments, could we improve the name?  Maybe `processIncrementOrScore` ...or something better?